webpack/containers/FormShowContainer.js

```diff
@@ -16,8 +16,6 @@ import { publishersProps } from "../prop-types/publisher_props";
 class FormShowContainer extends Component {
   componentWillMount() {
     this.props.fetchForm(this.props.params.formId);
-    this.props.fetchQuestions();
-    this.props.fetchResponseSets();
```

Comment:
When I do a complete refresh of a form show page, sometimes I get a permanent 'Loading...' in the response set list for the questions. Other times it works and the response set is in there. I wonder if removing this code is why i'm seeing this.